source/cloud/nvidia/brev.md

```diff
@@ -166,6 +166,9 @@ print(gdf)
 - Please note: Git is not preinstalled in the RAPIDS container, but can be installed into the container when it is running using
 
 ```bash
-apt update
-apt install git -y
+$ apt update
+```
+
+```bash
```

Comment:
This could be a multi-line block. Was there a reason for splitting it up?

Reply:
The problem with that is that the copy button copies everything and in my case (a personal preference) I rather have the choice of executing one command at the time. 

But I can revert it if you have a strong preference for executing them all at once.